src/graphnet/models/graphs/graphs.py

```diff
@@ -54,3 +54,41 @@ def __init__(
             perturbation_dict=perturbation_dict,
             seed=seed,
         )
+
+
+class IsolatedNodes(GraphDefinition):
+    """A Graph representation where each node is isolated."""
```

Comment:
Perhaps it would be more informative if we changed the doc string to say 

```
"""A Data representation without edge assignment.

I.e the resulting representation is created without an EdgeDefinition."""
```